app/src/main/java/com/duckduckgo/app/browser/omnibar/OmnibarLayout.kt

```diff
@@ -403,7 +385,7 @@ class OmnibarLayout @JvmOverloads constructor(
         }
     }
 
-    private fun render(viewState: ViewState) {
+    open fun render(viewState: ViewState) {
```

Comment:
the new experimental omnibar uses this open method to add some fuctionality for the mini-bar. doing it like this allows us to not change the current omnibar, and only add new things on top of that if necessary.